server/server-config.cpp

```diff
@@ -55,6 +55,9 @@ int ServerConfig::init_from_config(const char *config_path) noexcept {
     if (auto err_msg = set_cluster_name(cluster_name.data(), false)) {
       throw std::runtime_error(err_msg);
     }
+    if (auto environment = node["environment"]) {
+      environment_ = environment.as<std::string>();
```

Comment:
Maybe set it to some default value or something? Most likely we want to find servers for which `environment` is not specified

Reply:
We can do this by filtering by `empty` environment tag